packages/calcite-components/src/components/input-date-picker/input-date-picker.e2e.ts

```diff
@@ -1958,6 +1958,193 @@ describe("calcite-input-date-picker", () => {
   });
 });
 
+describe("proximitySelectionDisabled", () => {
+  it("should reset range and select a new start date on third click", async () => {
+    const page = await newE2EPage();
+    await page.setContent(
+      html`<calcite-input-date-picker range proximity-selection-disabled min="2099-01-01"></calcite-input-date-picker>`,
+    );
+    await skipAnimations(page);
+    await page.waitForChanges();
```

Comment:
`waitForChanges` shouldn't be necessary after test setup (`setContent` does this internally). Applies to other tests.